index.html

```diff
@@ -191,6 +191,32 @@ <h3 class="about-languajes--heading">Languajes</h3>
             </ul>
         </div>
     </section>
+    <section class="form-section">
+        <img src="/img/form-rectangle.png" alt="" class="form-rectangle">
+        <img src="/img/form-half-circle.png" alt="" class="form-half-circle">
+        <img src="/img/form-elipse.png" alt="" class="form-img1">
+        <h2 class="form-heading">
+            Contact me
+        </h2>
+        <p class="form-paragraph">If you have an application you <br> are interested in developing, a <br> feature that
+            you need
+            built or a <br> project that needs coding. I’d <br> love to help with it </p>
+
+        <form class="container" action="https://formspree.io/f/mwkywowd" method="post">
+            <fieldset>
+                <input class="text" type="text" name="name" placeholder="Maury" maxlength="30" required>
+                <input class="email" type="email" name="email" placeholder="maury94@gmail.com" required>
+                <textarea class="text-area" name="message" cols="30" rows="10"
+                    placeholder="Write your message here" maxlength="500" required></textarea>
+                <button type="submit" class="submit">
```

Comment:
- [ ] nice implementing button style effect on hover but you should also implement it when the button is pressed so please try to do so